Jenkinsfile

```
@@ -685,7 +686,16 @@ pipeline {
                             steps {
                                 powershell '''
                                 $ErrorActionPreference = 'Stop'
-                                .\\hack\\ci\\windows.ps1
+                                [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+                                Invoke-WebRequest https://github.com/jhowardmsft/docker-ci-zap/blob/master/docker-ci-zap.exe?raw=true -OutFile C:/Windows/System32/docker-ci-zap.exe
+                                $env:SOURCES_DRIVE="d"
+                                $env:SKIP_VALIDATION_TESTS="true"
```

Can you use `1` here? It's what we do elsewhere

```suggestion
                                $env:SKIP_VALIDATION_TESTS="1"
```